### PR TITLE
Add jsonapi-requests to conda-forge.

### PR DESCRIPTION
**Pull request by sodre**
Dependencies:
- [x] Tenacity (included in the PR).
- [x] Monotonic 1.3 (waiting for  conda-forge/monotonic-feedstock#7)

### CODE REVIEW

File: recipes/monotonic/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+{% set name = "monotonic" %}
+{% set version = "1.3" %}
```
Comment (isuruf) [inline recipes/monotonic/meta.yaml]: `monotonic` is already in `conda-forge`. You can submit a PR to update to 1.3

Comment (sodre) [inline recipes/monotonic/meta.yaml]: that was silly of me. I will do that right away.
I was essentially in auto-mode of condo-build -> fail -> add recipe -> repeat.

Thanks for catching it.

**Comment by sodre** [comment]
@isuruf jsonapi-requests is now ready for review.

File: recipes/jsonapi-requests/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+build:
+  number: 0
+  skip: True  # [py27]
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - pip
+    - pytest-runner
```
Comment (isuruf) [inline recipes/jsonapi-requests/meta.yaml]: Is this really needed?

Comment (sodre) [inline recipes/jsonapi-requests/meta.yaml]: unfortunately yes, they are importing pip directly in setup.py if I recall correctly.

Comment (isuruf) [inline recipes/jsonapi-requests/meta.yaml]: I mmeant `pytest-runner`

Comment (sodre) [inline recipes/jsonapi-requests/meta.yaml]: Yes, you can see it in the original [setup.py line 15](https://github.com/socialwifi/jsonapi-requests/blob/98d408abae8047c566d4fbfba583c25524efa70f/setup.py#L15)

**Pull request merged**